packages/flutter/test/widgets/html_element_view_test.dart

```diff
@@ -37,6 +37,36 @@ void main() {
       );
     });
 
+    testWidgets('Create HTML view with PlatformViewCreatedCallback', (WidgetTester tester) async {
+      final int currentViewId = platformViewsRegistry.getNextPlatformViewId();
+      final FakeHtmlPlatformViewsController viewsController = FakeHtmlPlatformViewsController();
+      viewsController.registerViewType('webview');
+
+      void onPlatformViewCreatedCallBack(int id) {
+        expect(id >= 0, true);
```

Comment:
I am not sure I follow this test, it there something this callback can do that we can validate after the fact rather than just expect a value? If it is not being called, it won't actually fail the test.

Reply:
But how to expect the callback to be called?

Reply:
The callback can set a boolean variable defined outside the callback to true and then you can check that the variable was in fact set to true.